app/src/main/res/layout/main_fragment.xml

```diff
@@ -14,72 +14,67 @@
+            android:layout_width="wrap_content"
+            android:layout_height="0dp"
+            android:layout_marginTop="8dp"
+            app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintHorizontal_bias="0.166"
-            app:layout_constraintStart_toEndOf="@+id/btnCatagories"
-            tools:layout_editor_absoluteY="105dp" />
+            app:layout_constraintHorizontal_bias="1.0"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/txtShowing"
+            app:layout_constraintVertical_bias="0.0" />
 
         <TextView
             android:id="@+id/txtShowing"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_marginTop="10dp"
-            android:layout_marginBottom="10dp"
-            android:text="Showing:    Recipes"
-            app:layout_constraintBottom_toTopOf="@+id/lstRecipes"
-            app:layout_constraintTop_toBottomOf="@+id/btnCatagories"
+            android:layout_marginTop="8dp"
             android:gravity="center_horizontal"
-             />
+            android:text="@string/selectaRecipe"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/linearLayout" />
+
+        <LinearLayout
+            android:id="@+id/linearLayout"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/rcpSearch">
+
+                <ImageButton
+                    android:id="@+id/btnShoppingList"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="1"
+                    android:contentDescription="@string/shoppingListPage"
+                    android:src="@android:drawable/ic_menu_sort_by_size" />
+
+                <ImageButton
+                    android:id="@+id/btnCategories"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="1"
+                    android:contentDescription="@string/categories"
+                    android:src="@android:drawable/ic_dialog_dialer" />
+
+                <ImageButton
+                    android:id="@+id/btnSchedule"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="1"
+                    android:contentDescription="@string/schedule"
+                    android:src="@android:drawable/ic_menu_month" />
+        </LinearLayout>
```

Comment:
Use the linear layout to format the three image buttons we created, so they will look much cleaner.